js/libs/form.js

```diff
@@ -506,7 +506,8 @@ Fliplet.Widget.instance('form-builder', function(data) {
           $vm.fields.forEach(function(field) {
             if (field.mediaFolderId) {
               formData._flSchema[field.name] = {
-                mediaFolderId: field.mediaFolderId
+                mediaFolderId: field.mediaFolderId,
+                name: field.name + ' ' + moment().format('YYYY-MM-DD HH:mm')
```

Comment:
```suggestion
                name: field.name + ' ' + moment().format('YYYY-MM-DD HH:mm') + '.png'
```

See https://github.com/Fliplet/fliplet-studio/issues/6818#issuecomment-662521923 We need the file extension added as well. Can you confirm if this works?

Reply:
It does work. Fixed.